### PR TITLE
Repositories

### PR DESCRIPTION
**Pull request by LolaOmolambe**
- Added repositories

### CODE REVIEW

File: src/main/java/com/appraisal/repositories/EmployeeManagerRepository.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.appraisal.repositories;
+
+import com.appraisal.entities.EmployeeManager;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```
Comment (johnojetunde) [inline src/main/java/com/appraisal/repositories/EmployeeManagerRepository.java]: You don't need this. Spring will automatically scan this as long as it extends JpaRepository/CrudRepository

File: src/main/java/com/appraisal/repositories/FeedbackRequestEntryRepository.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.appraisal.repositories;
+
+import com.appraisal.entities.FeedbackRequestEntry;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```
Comment (johnojetunde) [inline src/main/java/com/appraisal/repositories/FeedbackRequestEntryRepository.java]: ditto

File: src/main/java/com/appraisal/repositories/EmployeeTeamRepository.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package com.appraisal.repositories;
+
+import com.appraisal.entities.EmployeeTeam;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```
Comment (johnojetunde) [inline src/main/java/com/appraisal/repositories/EmployeeTeamRepository.java]: ditto

File: src/main/java/com/appraisal/repositories/EmployeeRepository.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package com.appraisal.repositories;
+
+import com.appraisal.entities.Employee;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```
Comment (johnojetunde) [inline src/main/java/com/appraisal/repositories/EmployeeRepository.java]: ditto

File: src/test/resources/application-test.properties [hunk 5]
```diff
@@ -1,4 +1,4 @@
-spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.hibernate.ddl-auto=create
```
Comment (johnojetunde) [inline src/test/resources/application-test.properties]: why did you change this?

**Review by johnojetunde — CHANGES_REQUESTED** [review]

File: src/test/resources/application-test.properties (hunk 5 above)
Comment (LolaOmolambe) [reply src/test/resources/application-test.properties]: I was just trying different things, would switch back

**Review by johnojetunde — APPROVED** [review]
